packages/fauna/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@next-auth/fauna-adapter",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Bhanu Teja P",
+  "license": "ISC",
+  "peerDependencies": {
+    "next-auth": "^3.1.0",
+    "faunadb": "^4.0.2"
+  },
+  "dependencies": {
+    "crypto": "^1.0.1"
```

Comment:
From npm:
> This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.

In other words, I think it is unnecessary to add it as a dependency.